_inc/lib/icalendar-reader.php

```diff
@@ -96,7 +96,7 @@ function apply_timezone_offset( $events ) {
 		} else {
 			// If the timezone isn't set then the GMT offset must be set.
 			// generate a DateInterval object from the timezone offset
-			$gmt_offset = get_option( 'gmt_offset' ) * HOUR_IN_MINUTES;
+			$gmt_offset = get_option( 'gmt_offset' ) * HOUR_IN_SECONDS;
 			$timezone_offset_interval = date_interval_create_from_date_string( "{$gmt_offset} minutes" );
```

Comment:
Will this cause any problems with line 100 where it is getting the time difference in minutes? Should that be changed to seconds?
The HOUR_IN_MINUTES constant returns 60, and the HOUR_IN_SECONDS constant returns 3600.